typewise-alert.js

```diff
@@ -1,54 +1,60 @@
+const COOLING_LIMITS = {
+  PASSIVE_COOLING: {lowerLimit: 0, upperLimit: 35},
+  HI_ACTIVE_COOLING: {lowerLimit: 0, upperLimit: 45},
+  MED_ACTIVE_COOLING: {lowerLimit: 0, upperLimit: 40},
+};
 
 function inferBreach(value, lowerLimit, upperLimit) {
   if (value < lowerLimit) {
     return 'TOO_LOW';
-  }
-  if (value > upperLimit) {
+  } else if (value > upperLimit) {
     return 'TOO_HIGH';
   }
   return 'NORMAL';
 }
 
-function classifyTemperatureBreach(coolingType, temperatureInC) {
-  let lowerLimit = 0;
-  let upperLimit = 0;
-  if (coolingType == 'PASSIVE_COOLING') {
-    lowerLimit = 0;
-    upperLimit = 35;
-  } else if (coolingType == 'HI_ACTIVE_COOLING') {
-    lowerLimit = 0;
-    upperLimit = 45;
-  } else if (coolingType == 'MED_ACTIVE_COOLING') {
-    lowerLimit = 0;
-    upperLimit = 40;
-  }
+function classifyTemperatureBreach(coolingType, temperatureInC, limits) {
+  const {lowerLimit, upperLimit} = limits[coolingType];
   return inferBreach(temperatureInC, lowerLimit, upperLimit);
 }
 
 function checkAndAlert(alertTarget, batteryChar, temperatureInC) {

```

Comment:
test code for checkAndAlert not found ? for the current implementation how one can write Unit Test. code? checkAndAlert coupled with actual implementation of dependencies (sentToController and sentToEmail)